eng/pipelines/windows.yml

```diff
@@ -115,14 +115,27 @@ stages:
         buildExtraArguments: /p:RuntimeOS=win10
 
         variables:
-          - uapNetfxQueues: Windows.10.Amd64.ClientRS5.Open
-          - windowsArmQueue: Windows.10.Arm64.Open
+          - ${{ if eq(parameters.isInternalBuild, 'false') }}:
+            - nanoQueues: "`(Windows.Nano.1809.Amd64.Open`)windows.10.amd64.serverrs5.open@mcr.microsoft.com/dotnet-buildtools/prereqs:nanoserver-1809-helix-amd64-08e8e40-20200107182504"
```

Comment:
I like that the nanoQueues definition is back; but to actually restore Nano we need to undo the two other edits from

https://github.com/dotnet/corefx/pull/43136/commits/8ebe7bbe421fa2fbcfcf754cfbe5979fffa2863c

(Lines 38 and 64)